teacher.rb

```diff
@@ -0,0 +1,10 @@
+module Teacher
+  def initialize(specialization)
+    @specialization = specialization
+    super
```

Comment:
here you should also initialize age, name = "Unknown" and parent_permission = true
and add its super classes 😄